accumulator_updater/programs/accumulator_updater/src/state/accumulator_input.rs

```diff
@@ -14,26 +14,94 @@ use {
 /// the CPI calling program (e.g. Pyth Oracle)
 ///
 /// TODO: implement custom serialization & set alignment
-#[account]
+#[account(zero_copy)]
+#[derive(Debug, InitSpace)]
```

Comment:
nitpick for a future PR: can we rename this thing? It's more like a `MessageBuffer` or something right? It's a thing that stores messages. Those messages happen to be accumulated by the validator, but that doesn't have to play a role in the name of this struct.

(I think we're using the term "accumulator" way too much throughout our code / conversations right now. Everything is an accumulator X or accumulator Y, which makes it hard to distinguish the different concepts)